backend/taxHandlers.ts

```diff
@@ -0,0 +1,102 @@
+import { check } from "prettier";
+import { TaxForOrderRes } from "taxjar/dist/util/types";
+import { OrderSubscriptionFragment } from "../generated/graphql";
+import { createOrderTransaction, fetchTaxes } from "./taxjarApi";
+import {
+  CheckoutPayload,
+  ResponseTaxPayload,
+  FetchTaxesPayload,
+  TaxJarConfig,
+  OrderPayload,
+} from "./types";
+
+const _calculateTaxes = async (
+  taxData: FetchTaxesPayload,
+  taxJarConfig: TaxJarConfig
+): Promise<{ data: ResponseTaxPayload }> => {
+  const taxResposne = await fetchTaxes(taxData, taxJarConfig);
+  const taxDetails = taxResposne.tax.breakdown;
+  const shippingDetails = taxDetails?.shipping;
+  let shippingPriceGross = Number(taxData.shipping_amount);
+  let shippingPriceNet = Number(taxData.shipping_amount);
+  let shippingTaxRate = 0;
+
+  if (taxDetails !== undefined) {
+    if (shippingDetails !== undefined) {
+      shippingPriceGross =
+        shippingDetails.taxable_amount + shippingDetails.tax_collectable;
+      shippingPriceNet = shippingDetails.taxable_amount;
+      shippingTaxRate = shippingDetails.combined_tax_rate;
+    }
+  }
```

Comment:
`shippingDetails` is derived from `taxDetails`
```suggestion
  if (shippingDetails) {
      shippingPriceGross =
        shippingDetails.taxable_amount + shippingDetails.tax_collectable;
      shippingPriceNet = shippingDetails.taxable_amount;
      shippingTaxRate = shippingDetails.combined_tax_rate;
  }
```